### ISSUE TITLE
convert .val/.vit file  to svg file programmatically 

### ISSUE THREAD

**Issue by marinkreso95** [FabriceSalvaire/Patro#35]
Is there an option to convert val/vit file to SVG (or pdf) file with Python function. 

I see we have SvgFileWriter but I don't understand nor can I find an example how and what arguments to pass from vit/val files to the constructor of SvgFileWriter. 

Is this even possible? If yes, could you please show some example?

Otherwise, your library is really great.

**Comment by dismine** [comment]
Yes, it would be great if it was 100% compatible with Valentina (not to be confused with S2D). God, I miss Python implementation. For now, what we do, it is to use Valentina itself to export pattern in SVG.
